integration-test-groups/cxf-soap/cxf-soap-ssl/src/main/resources/application.properties

```diff
@@ -0,0 +1,29 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+quarkus.cxf.path=/soapservice
+quarkus.native.resources.includes=truststore-client.jks,truststore-server.jks,truststore-wrong.jks
+
+# Workaround for https://github.com/apache/camel-quarkus/issues/4603
+# Should not be needed with the Quarkus release coming after 3.0.0.Alpha5
+quarkus.jaxb.validate-jaxb-context = false
```

Comment:
```suggestion
```

Could you please remove this? It is not needed anymore with Quarkus 3.0.0.Alpha6.

Reply:
Thanks! Removed.